docs/index.rst

```diff
@@ -633,6 +641,10 @@ End clients (non-CP clients):
 
          kafka1            | 	super.users = User:admin;User:mds;User:superUser;User:ANONYMOUS
 
+#. From the |c3| UI, in the Administration menu, click the *Manage role assignments* option. Click on ``Assignments`` and then the Kafka cluster ID. From the ``Topic`` list, verify that the LDAP user ``appSA`` is allowed to access a few topics, including any topic whose name starts with ``wikipedia``. This role assignment was done during ``cp-demo`` bringup in the :devx-cp-demo:`create-role-bindings.sh script|scripts/helper/create-role-bindings.sh`.
```

Comment:
> during cp-demo bringup ...

Maybe "startup" instead of "bringup" would read more clearly?

Reply:
@ybyzek I modified this bit based on @mikebin 's feedback and pushed a change to this branch.  I also slightly modified the associated doc image to remove some extraneous pixels around the border.

Reply:
Thanks @rspurgeon !